packages/@aws-cdk/aws-rds/lib/instance-parameter-group-ref.ts

```diff
@@ -0,0 +1,44 @@
+import { CfnOutput, Construct } from "@aws-cdk/cdk";
+
+/**
+ * A instance parameter group
+ */
+export abstract class InstanceParameterGroupRef extends Construct {
```

Comment:
We stopped using the `Ref` suffix an instead use interfaces. Lambda is a good example:

https://github.com/awslabs/aws-cdk/blob/0de2206f8ba324fbfe0a3685eb5a406d17f8a422/packages/%40aws-cdk/aws-lambda/lib/function-base.ts#L14

https://github.com/awslabs/aws-cdk/blob/0de2206f8ba324fbfe0a3685eb5a406d17f8a422/packages/%40aws-cdk/aws-lambda/lib/function-base.ts#L126

https://github.com/awslabs/aws-cdk/blob/0de2206f8ba324fbfe0a3685eb5a406d17f8a422/packages/%40aws-cdk/aws-lambda/lib/function.ts#L214

https://github.com/awslabs/aws-cdk/blob/0de2206f8ba324fbfe0a3685eb5a406d17f8a422/packages/%40aws-cdk/aws-lambda/lib/function.ts#L564

Reply:
These are imported but not used yet, so just removed them. fixed db1c8f3